source/Cosmos.System2/Power.cs

```diff
@@ -1,11 +1,23 @@
-﻿namespace Cosmos.System
+﻿using sysIO = System.IO;
```

Comment:
Not sure about if this addition can break anything.
would love to hear your thoughts.

Reply:
I dont really get the reason for this addition.

Reply:
Cosmos.System shadows System, therefore IOException is unknown when documenting it.
I use this reference in line 20:
<exception cref="sysIO.IOException">Thrown on IO error.</exception>.

I can remove this line and metion System.IO.IOException, but its unknown type, as I mentioned.